src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -457,7 +461,7 @@ const FileField = props => {
                 } else {
                   focusElement('.usa-input-error, .input-error-date, [error]');
                 }
-              }, 100);
+              }, 250);
```

Comment:
Bumped this timeout as initial focus while using `scrollAndFocusTarget` callback would be executed after this and maintain focus on the `H3`.